### ISSUE TITLE
Doesn't work on Ubuntu

### ISSUE THREAD

**Issue by vadi2** [bobbyg603/myQtCrasher#14]
When I start the example, stderr says `[317443:317443:20211116,143425.584992:ERROR filesystem_posix.cc:57] /home/vadi/Downloads/build-myQtCrasher-Desktop_Qt_5_15_2_GCC_64bit-Debug/crashpad not a directory` - and that is true, at that location is a binary.

When I crash the application, nothing gets reported to https://app.bugsplat.com/v2/crashes?database=Fred&c0=appName&f0=EQUAL&v0=myQtCrasher - I'm guessing due to the above error.

What could be the fix?

**Comment by bobbyg603** [comment]
@vadi2 I believe there’s a build step that’s supposed to copy `crashpad-handler` to the output directory. Maybe this step has rusted? You might fiddle with the scripts in the .pro file until we get around to looking at this.

**Comment by bobbyg603** [comment]
@vadi2 try modifying [line 97](https://github.com/BugSplat-Git/my-qt-crasher/blob/648859c3d1eb33dcbf5486dde0ea912c3cdd0888/myQtCrasher.pro#L97) 

```
QMAKE_POST_LINK += "mkdir -p $$OUT_PWD/crashpad && cp $$PWD/Crashpad/Bin/Linux/crashpad_handler $$OUT_PWD/crashpad/crashpad_handler"
```

at some point in the past this step used to work the way it was written...

**Comment by vadi2** [comment]
Yep precisely, I arrived at the same conclusion as well.

**Comment by bobbyg603** [comment]
@vadi2 thanks for alerting us of the problem - just merged a fix (#15) 🚀
